### DISCUSSION TITLE
Entry Point Not Found

### DISCUSSION THREAD

**Discussion by Abdollah0011** [msasanmh/DNSveil#214] [Q&A]
![Untitled](https://github.com/user-attachments/assets/06013329-600e-4c20-ac79-8e85081c88e1)
Hello, I get this error at the beginning of entering the software, and then DNS does not connect.

**Comment by msasanmh** [comment]
> ![Untitled](https://github.com/user-attachments/assets/06013329-600e-4c20-ac79-8e85081c88e1)
> Hello, I get this error at the beginning of entering the software, and then DNS does not connect.

1. Uninstall the application. 
2. Install or update .NET 6 Desktop and .NET 6 ASP Core Runtime. 
3. Download the application from GitHub (Official Source) and install it.
4. Exclude the app and all executable files in "binary" directory in your Antivirus or any Security Software you have. 
5. Run the application.

**Comment by msasanmh** [comment]
Also, read this:
https://www.quora.com/How-do-I-fix-an-entry-point-not-found-error/answer/Wayne-Sue-2
